### ISSUE TITLE
lg-remote-control not working on HASSIO 2021.1.4

### ISSUE THREAD

**Issue by kalosjann** [madmicio/LG-WebOS-Remote-Control#44]
Hello, 
I'm facing challenges with the lg-remote-control script on HA 2021.1.4 (I've installed it through the IOTStack project)
I'm getting the following error:
<img width="297" alt="Screenshot 2021-01-23 at 09 14 09" src="https://user-images.githubusercontent.com/11614499/105573077-5eded100-5d5b-11eb-8061-db221cc9dc13.png">

Basically I've placed the lg-remote-control.js script inside the following folder:
/usr/share/hassio/homeassistant/config/www/ 
and then added it into the Lovelace resources as
/local/lg-remote-control.js

Please note that I followed the same approach for another js custom components and it working fine.
Can someone help me?

Thanks

**Comment by swap83** [comment]
yes. Me too. Hassio 2021.2.2

**Comment by kalosjann** [comment]
I finally solved the issue.
I managed to install the extension through HACS, and it worked at the first attempt.

**Comment by swap83** [comment]
In the section: lovelace config: default view:
There is a - at beginning of the text. Without - the extension works!

**Comment by SkywalkerPTE419** [comment]
> In the section: lovelace config: default view:
> There is a - at beginning of the text. Without - the extension works!

Yes that resolved my problem, i just copy/past the example and after that remove the extra "-"
